projects/plugins/jetpack/extensions/blocks/ai-chat/components/nudge-enable-search/index.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+/*
+ * Internal dependencies
+ */
+import { Nudge } from '../../../../shared/components/upgrade-nudge';
+import useAutosaveAndRedirect from '../../../../shared/use-autosave-and-redirect';
+
+const EnableJetpackSearchPrompt = () => {
+	const checkoutUrl = `${ window?.Jetpack_Editor_Initial_State?.adminUrl }admin.php?page=jetpack-search`;
+	const { autosaveAndRedirect, isRedirecting } = useAutosaveAndRedirect( checkoutUrl );
+
+	if ( window?.Jetpack_Editor_Initial_State?.search?.instant_search_enabled ) {
+		return null;
+	}
+
+	const goToCheckoutPage = event => {
+		autosaveAndRedirect( event );
+	};
+
+	return (
+		<Nudge
+			buttonText={ __( 'Enable Jetpack Search', 'jetpack' ) }
+			checkoutUrl={ checkoutUrl }
+			className={ 'jetpack-ai-connect-banner' }
+			description={ __(
+				'You need to enable Jetpack Search so that Jetpack AI can index your site.',
+				'jetpack'
+			) }
+			goToCheckoutPage={ goToCheckoutPage }
```

Comment:
Can we rely on `useModuleStatus` from `@automattic/jetpack-shared-extension-utils` to activate the module straight from the editor?

Reply:
That is something we probably want to look into later, as specifics to which plan this belongs are still ironed out. We want to show some information ASAP

Reply:
Yeah we'll need to come back to this as we haven't yet settled on whether or which the plan will be required or not. If so, we'll want it trigger a plan purchase flow.